src/rgw/store/sfs/types.h

```diff
@@ -162,10 +165,18 @@ class Bucket {
     deleted.insert(objref);
     objects.erase(it);
   }
+
+  inline std::string get_cls_name() { return "sfs::bucket"; }
 };
 
 using BucketRef = std::shared_ptr<Bucket>;
 
+using MetaBucketsRef = std::shared_ptr<sqlite::SQLiteBuckets>;
+
+static inline MetaBucketsRef get_meta_buckets(sqlite::DBConnRef conn) {
+  return std::make_shared<sqlite::SQLiteBuckets>(conn);
+}
+
 }  // ns rgw::sal::sfs
 
 #endif // RGW_STORE_SFS_TYPES_H
```

Comment:
no new line

Also... the ObjectState enum could be placed in this file and remove the extra file just for it... 
I think it makes sense, as this file would have all the types used by `sfs`

Reply:
While I agree with your point, I think this is something we can handle when we work on versioning.